poucave/html/js/app/index.mjs

```diff
@@ -1,11 +1,29 @@
 import { html, render } from "../htm_preact.mjs";
 
 import App from "./components/App.mjs";
+import { ROOT_URL } from "./constants.mjs";
 
 // Initialize the app on page load
-window.addEventListener("load", () => {
+window.addEventListener("load", async () => {
   // Clear the application container and render the application
   const appContainer = document.getElementById("app");
   appContainer.innerHTML = "";
-  render(html`<${App} />`, appContainer);
+
+  // Fetch instance metadata.
+  const resp = await fetch(new URL("/", ROOT_URL));
+  const { service } = await resp.json();
+  const title = capitalize(service);
```

Comment:
This worries me. I think it makes sense to use the capitalized service as the title default, but the service is used elsewhere, and it makes sense to me to disconnect the human-visible page title from a service name that may be used in links and APIs.